packages/react-hooks/package.json

```diff
@@ -28,13 +28,13 @@
     "rxjs": "^7.2.0"
   },
   "devDependencies": {
-    "@aries-framework/core": "^0.2.0",
+    "@aries-framework/core": "^0.2.5-alpha.3",
```

Comment:
We keep extensions aligned with the stable releases. If you want this feature available in an extension we can make a new release

Reply:
Let me know if you want me to make a new release

Reply:
That would be amazing if we could make a new release with the connection types and use that, let us know if there's anything we can do to help with that